yarpl/include/yarpl/Refcounted.h

```diff
@@ -176,35 +181,124 @@ bool operator==(const Reference<T>& lhs, const Reference<U>& rhs) noexcept {
   return lhs.get() == rhs.get();
 }
 
+template <typename T>
+bool operator==(const Reference<T>& lhs, std::nullptr_t) noexcept {
```

Comment:
Are these overloads actually necessary?  I would figure `someRef == nullptr` would just forward to `operator==(const Reference<T>&, const Reference<T>&)` because `Reference<T>` has an implicit ctor for `std::nullptr_t`.